tests/fixtures/config_or_event.json

```diff
@@ -0,0 +1,12 @@
+{
+  "database" : "prod-test-db",
+  "state_doc_bucket" : "bucket-to-hold-state-documents",
+  "snapshot_region" : "us-west-1",
+  "checks" : [
```

Comment:
You might want to skip the checks for this first version.  It might be getting a little too in the weeds at the moment, and adding complexity.  I'd go with an MVP for now - lets just get the database restored :)